src/cdk/table/table.scss

```diff
@@ -1,3 +1,17 @@
 .cdk-table-fixed-layout {
   table-layout: fixed;
 }
+
+.cdk-table-virtual-scroll {
+  border-collapse: collapse;
```

Comment:
Add a comment for why `border-collapse` is important here?

Reply:
Tables with multiple headers/footers that have `border-spacing` will initially render with spacing between cells. However, the spacing collapses after scrolling. I'm able to reproduce this with non-virtualized CDK tables, so I believe this is an issue with the stickystyler.

I added this style so border-spacing wouldn't collapse on scroll. It can probably be removed since it's an existing behavior with the sticky styler.